mkdocs.yml

```diff
@@ -15,7 +15,6 @@ plugins:
     - search
     - redirects:
         redirect_maps:
```

Comment:
I didn't remove this entirely because of https://github.com/martinvonz/jj/pull/2312#discussion_r1343537918.